pages/how-to-contribute.md

```diff
@@ -0,0 +1,157 @@
+1. Push any changes from the review to the branch as new commits so the reviewer only needs to review new changes. Please avoid squashing commits after the review starts. Squashing makes it hard for the reviewer to follow the changes.
+1. Repeat this process until a reviewer approves the pull request.
+1. When the review process is finished, all commits on the pull request may be squashed by a committer. Please avoid squashing as it makes it difficult for the committer to know if they are merging what was reviewed.
+
+## Coding Guidelines
+
+* If a change needs to go into multiple branches of Accumulo, it should be merged into earlier branches then later branches. 
+* Accumulo follows [semver] for its public API. Accumulo lists which packages are public API in its [README.md][accumulo-readme]. 
+* Every file requires the ASF license header as described in [ASF Source Header][srcheaders].
+* Remove all trailing whitespaces. Eclipse users can use Source&rarr;Cleanup option to accomplish this.
+* Use 2 space indents and never use tabs!
+* Use 160-column line width for Java code and Javadoc.
+* Use a new line with single statement if/else blocks.
+* Do not use Author Tags. The code is developed and owned by the community.
+
+## Code Editors
+
+Feel free to use any editor when contributing Accumulo. If you are looking for a recommendation, many Accumulo
+developers use [IntelliJ][intellij] or [Eclipse][eclipse]. Below are some basic instructions to help you get started.
+
+### IntelliJ
+
+1. Download and install [IntelliJ][intellij]
+1. Clone the Accumulo repository that you want to work on.
+   ```shell
+   git clone https://github.com/apache/accumulo.git
+   ```
+1. [Import][intellij-import] the repository as a Maven project into IntelliJ
+1. (Optional) Download and import `Eclipse-Accumulo-Codestyle.xml` from Accumulo's [contrib][accumulo-contrib] directory
+  * Import via `File` > `Settings` > `Code Style` and clicking on cog wheel
+
+### Eclipse
+
+1. Download and install [Eclipse][eclipse].
+1. Clone the Accumulo repository that you want to work on.
+   ```shell
+   git clone https://github.com/apache/accumulo.git
+   ```
+1. [Import][eclipse-import] the repository as a Maven project into Eclipse
+1. (Optional) Download and import Eclipse formatting and style guides from Accumulo's [contrib][accumulo-contrib] directory
+  * Import Formatter: `Preferences` > `Java` > `Code Style` > `Formatter` and import the `Eclipse-Accumulo-Codestyle.xml` downloaded in the previous step. 
+  * Import Template: `Preferences` > `Java` > `Code Style` > `Code Templates` and import the `Eclipse-Accumulo-Template.xml`. Make sure to check the "Automatically add comments" box. This template adds the ASF header and so on for new code.
+
+## Helpful Links
+
+* **Build resources** - [TravisCI] & [Jenkins][jenkins] ([Master][masterbuild], [1.8 Branch][18build], [1.7 Branch][17build])
+* **Development** - [Building Accumulo][build], [Git Workflow][git]
```

Comment:
The **Development** and **Project governance** links could be removed as they are linked to in contributor's guide.